discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -0,0 +1,404 @@
+import logging
+import time
+import sqlalchemy as sa
+from src.tasks.celery_app import celery
+from src.utils.redis_constants import most_recent_indexed_aggregate_user_block_redis_key
+from src.tasks.calculate_trending_challenges import get_latest_blocknumber
+
+logger = logging.getLogger(__name__)
+
+# Names of the aggregate tables to update
+AGGREGATE_USER = "aggregate_user"
+DEFAULT_UPDATE_TIMEOUT = 60
+REFRESH_COUNTER = 10
```

Comment:
i'd make this higher like if we refresh this every 30 seconds, maybe we clear every 100 times? that's like roughly once an hour

Reply:
Yea, I was thinking this might've been too low, I'll raise it.